master/static/srv/get_checkpoints_for_experiment.sql

```diff
@@ -1,5 +1,6 @@
 SELECT
-    c.uuid AS uuid,
+    c.uuid::text AS uuid,
+    'STATE_' || c.state AS state,
```

Comment:
Maybe drop a comment that this is here because of how `QueryProto` works? Maybe not.